### PR TITLE
Adding previous releases to fbc catalog

### PR DESCRIPTION
**Pull request by OlivierCazade**
## Description

Adding previous releases to fbc catalog

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [ ] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labeled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: catalog/rc.yaml [hunk 1]
```diff
@@ -0,0 +1,1006 @@
+image:
```
Comment (jotak) [inline catalog/rc.yaml:1]: is this auto-generated?

Comment (OlivierCazade) [reply catalog/rc.yaml:1]: This was generated using opm migrate command to import an existing catalog

This was imported using the production Red Hat catalog, but now we have to manually maintain it.

File: catalog/rc.yaml [hunk 2]
```diff
@@ -0,0 +1,1006 @@
...
+    - email: rschaffe@redhat.com
+      name: Ronen Schaffer
+    - email: stlee@redhat.com
+      name: Steven Lee
+    maturity: stable
+    minKubeVersion: 1.23.0
+    provider:
+      name: Red Hat
+      url: https://www.redhat.com
+relatedImages:
```
[the remark this answers is not in the document]
Comment (OlivierCazade) [reply catalog/rc.yaml:996]: I did not have any security error there without them, but I can add them for clarity.

File: catalog/released/1.7.0.yaml [hunk 3]
```diff
@@ -0,0 +1,1011 @@
...
+- image: registry.redhat.io/network-observability/network-observability-flowlogs-pipeline-rhel9@sha256:28bdb6eae77da596c0def119c879e4fb3849065243fc804f4abf5480453b7b6d
+  name: flowlogs_pipeline
+- image: registry.redhat.io/network-observability/network-observability-operator-bundle@sha256:3a3c70c990bd3a012453857698433d674b98445e55cc9c5c4ed38eea29eb82b1
+  name: ""
+- image: registry.redhat.io/network-observability/network-observability-rhel9-operator@sha256:93328e2f6e95c9cce31e096c6591c91d419471ef13f441b3f20b16984f3f2f9d
+  name: network-observability-rhel9-operator-93328e2f6e95c9cce31e096c6591c91d419471ef13f441b3f20b16984f3f2f9d-annotation
+- image: registry.redhat.io/network-observability/network-observability-rhel9-operator@sha256:93328e2f6e95c9cce31e096c6591c91d419471ef13f441b3f20b16984f3f2f9d
+  name: manager
+- image: registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:e34f912158985525f6efb3c4b675f20854fc677c867d12fe3cd2e971309428dc
+  name: kube-rbac-proxy
```
[the remark this answers is not in the document]
Comment (OlivierCazade) [reply catalog/released/1.7.0.yaml:1010]: If it is listed there twice, I guess this is listed twice in the 1.7.0 release, I don't know why.

**Review by jotak — APPROVED** [review]

**Pull request merged**